remote-method/index.js

```diff
@@ -0,0 +1,244 @@
+'use strict';
+var yeoman = require('yeoman-generator');
+var chalk = require('chalk');
+
+var workspace = require('loopback-workspace');
+var ModelProperty = workspace.models.ModelProperty;
+
+var actions = require('../lib/actions');
+var helpers = require('../lib/helpers');
+var validateName = helpers.validateName;
+var accepts = [];
+var returns = [];
+
+module.exports = yeoman.generators.Base.extend({
+  // NOTE(bajtos)
```

Comment:
Please remove my name from the `NOTE` since it's not me who added it ;-) It's ok to keep the note itself though.